api-gateway/src/main/resources/static/css/petclinic.css

```diff
@@ -123,17 +123,21 @@ a.newBtn:hover, button#newBtn:hover{
   border: 1px solid black;
 }
 
+#bill{
+  border: 1px solid black;
+}
+
 table:not(.noColor) {
   background-color: white;
   color: black;
 }
 
-h2.titleOwner {
+/*h2.titleOwner {
```

Comment:
Did you check for usages of this CSS class across the rest of the service before removing it? (it's in a common class)

Reply:
Yes, I checked and no other services will be affected.